src/TransitorySensitivities.jl

```diff
@@ -178,10 +183,8 @@ end
 
 function normalizeparamtransitorysensitivities!(dSdt,ssys::SystemSimulation,t,ind)
     y = ssys.sol(t)
+    dSdt .*= ssys.p[ind]
     for sim in ssys.sims
-        if sim.domain.parameterindexes[1]+length(sim.species) < ind && sim.domain.parameterindexes[2] > ind
-            dSdt .*= ssys.p[ind]
-        end
         @views ns = y[sim.domain.indexes[1]:sim.domain.indexes[2]]
```

Comment:
Should revert this because we should only apply this correction to rate. The old way of implement will only happen once if the ind is within the particular simulation.

Reply:
Done